### PR TITLE
[Term Entry] PHP Math Functions round()

### LINKED ISSUE #2395: [Term Entry] PHP Math Functions round()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `round()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/round/round.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by LewisMK**
### Description

Closes #2395 

### Type of Change

- Adding a new entry

### Checklist

- [*] All writings are my own.
- [*] My entry follows the Codecademy Docs style guide.
- [*] My changes generate no new warnings.
- [*] I have performed a self-review of my own writing and code.
- [*] I have checked my entry and corrected any misspellings.
- [*] I have made corresponding changes to the documentation if needed.
- [*] I have confirmed my changes are not being pushed from my forked `main` branch.
- [*] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [*] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/round/round.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+Title: 'round()'
+Description: 'Round a number to the nearest whole number or to specified decimal numbers.'
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:3]:
```suggestion
Description: 'Returns the nearest whole number to the passed value, or the value rounded to the specified decimal precision.'
```

File: content/php/concepts/math-functions/terms/round/round.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`round()`** converts an integer or a float to the nearest whole number or to specified decimal numbers.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:16]:
```suggestion
The **`round()`** returns an integer or a float to the nearest whole number or to the specified decimal precision.
```

File: content/php/concepts/math-functions/terms/round/round.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Syntax
+
+```pseudo
+$roundedNumber = round($number);
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:21]:
```suggestion
$roundedNumber = round($number, $decimals, $roundMode);
```

Comment (caupolicandiaz) [reply content/php/concepts/math-functions/terms/round/round.md:21]: I recommend having one syntax block with all the parameters. And address each parameter below.

File: content/php/concepts/math-functions/terms/round/round.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+## Codebyte Example
+
+The codebyte below uses the `round()` function to round off `pi` to `3` decimal places:
+
+```codebyte/php
+<?php
+
+$originalNumber = 3.1415927;
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:69]:
```suggestion
$piConstant = pi();
```

File: content/php/concepts/math-functions/terms/round/round.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+$roundedNumber = round($originalNumber, 3);
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:71]:
```suggestion
$roundedNumber = round($piConstant, 3);
```

File: content/php/concepts/math-functions/terms/round/round.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+
+echo "Original Number: $originalNumber<br>";
+echo "Rounded Number: $roundedNumber";
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/round/round.md:74]:
```suggestion
echo "Original Number: $piConstant\n";
echo "Rounded Number: $roundedNumber";
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@LewisMK I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@LewisMK I made a couple of direct edits to the syntax section. This is ready for a second review.

**Comment by LewisMK** [comment]
I am truly sorry  @caupolicandiaz for not competing the PHP roundmode. I was held up trying to do some projects for a SOC-Analyst entry role. I am sorry for my tardiness.  I will avail myself better in soon time to come.

**Comment by LewisMK** [comment]
If you want me to complete it, I can manage and I will dedicate one day to finish immediately. Your suggestions helped me to learn about multiple PHP round modes that I never knew.

**Comment by LewisMK** [comment]
I am qualified and ready to take on SOC  analyst work...I know how to analyze malicious docs and traffic...If you have space for me to learn more for volunteer, I am willing. Thanks.

**Review by SSwiniarski — APPROVED** [review]
@LewisMK Looks good.

**Pull request merged**